pep-0660.rst

```diff
@@ -186,15 +192,73 @@ Frontends must not expose the wheel obtained from ``build_wheel_for_editable``
 to end users. The wheel must be discarded after installation and must not be
 cached nor distributed.
 
+Limitations
+===========
+
+This PEP does not support making the content of the wheel .data directory
+"editable" (scripts, C header files, and other data files).
```

Comment:
How about

This PEP focuses on making the ``purelib|platlib`` categories (installed into ``site-packages``) "editable".  It does not make special provision for the other categories ``headers|scripts|data``.  Package authors are encouraged to use ``console_scripts``, make their scripts tiny wrappers around library functionality, or manage these from the source checkout during development.

Because:

`distribution-1.0.data/(purelib|platlib|headers|scripts|data)`.

`(purelib|platlib|headers|scripts|data)` can all appear in the `.data` directories. These are called `install paths` or `categories`.

Reply:
Thanks. Update done.